app/services/proceeding_type_full_text_search.rb

```diff
@@ -39,7 +39,14 @@ def matching_results
   end
 
   def instantiate_result(row)
-    Result.new(row["meaning"].strip, row["ccms_code"], row["description"].strip, row["full_s8_only"], row["ccms_category_law"]&.strip, row["ccms_matter"])
+    Result.new(row["meaning"].strip,
+               row["ccms_code"],
+               row["description"].strip,
+               row["full_s8_only"],
+               row["sca_core"],
+               row["sca_related"],
+               row["ccms_category_law"]&.strip,
+               row["ccms_matter"])
```

Comment:
Although the majority of this is existing code, why are we only `stripping` some strings?

Reply:
I _think_ it's because those are the searchable fields, the others are either booleans or stored lookup values